domain/src/main/java/com/github/leuludyha/domain/model/user/preferences/UserPreferences.kt

```diff
@@ -1,8 +1,13 @@
 package com.github.leuludyha.domain.model.user.preferences
 
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+
 open class UserPreferences(
     /** Map WorkId -> WorkPreference */
-    var workPreferences: MutableMap<String, WorkPreference> = mutableMapOf()
+    var workPreferences: MutableMap<String, WorkPreference> = mutableMapOf(),
+    /** Whether this user prefers dark mode or not */
+    val darkTheme: MutableState<Boolean> = mutableStateOf(false),
```

Comment:
But this one should definitely not be stored in `Room`. Rather using `DataStore` (https://developer.android.com/topic/libraries/architecture/datastore)

Reply:
I'm not sure I get what you mean

Reply:
Not now, but later on we will store this information using `DataStore` to make it persistent
It shouldn't be saved in the `Room` database I think